config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml

```diff
@@ -420,3 +420,31 @@ presubmits:
               memory: "6Gi"
           securityContext:
             privileged: true
+
+  - name: pull-perf-tests-util-images-access-tokens
+    always_run: false
+    skip_report: true
+    max_concurrency: 10
+    run_if_changed: ^util-images/.*$
+    branches:
+      - master
+    labels:
+      preset-service-account: "true"
+      preset-dind-enabled: "true"
+    spec:
+      containers:
+        - image: gcr.io/k8s-testimages/bootstrap:v20200124-81ee414
+          args:
+          - --repo=k8s.io/$(REPO_NAME)=$(PULL_REFS)
+          - --root=/go/src
+          - --timeout=60
+          - --scenario=execute
+          - --
+          - /go/src/k8s.io/perf-tests/util-images/presubmit.sh
```

Comment:
Would it work if you used $GOPATH/src/... ?

Reply:
Unfortunately, no. $GOPATH is not substituted in [execute.py](https://github.com/kubernetes/test-infra/blob/bd3df30a22bbba5af63509fb930a382484e3d912/scenarios/execute.py).

There is an error
```
W0128 07:00:33.868] Run: ('$GOPATH/k8s.io/perf-tests/util-images/presubmit.sh',)
W0128 07:00:33.873] Traceback (most recent call last):
W0128 07:00:33.873]   File "/workspace/./test-infra/jenkins/../scenarios/execute.py", line 50, in <module>
W0128 07:00:33.874]     main(ARGS.env, ARGS.cmd + ARGS.args)
W0128 07:00:33.874]   File "/workspace/./test-infra/jenkins/../scenarios/execute.py", line 41, in main
W0128 07:00:33.874]     check(*cmd)
W0128 07:00:33.874]   File "/workspace/./test-infra/jenkins/../scenarios/execute.py", line 30, in check
W0128 07:00:33.875]     subprocess.check_call(cmd)
W0128 07:00:33.875]   File "/usr/lib/python2.7/subprocess.py", line 185, in check_call
W0128 07:00:33.875]     retcode = call(*popenargs, **kwargs)
W0128 07:00:33.876]   File "/usr/lib/python2.7/subprocess.py", line 172, in call
W0128 07:00:33.876]     return Popen(*popenargs, **kwargs).wait()
W0128 07:00:33.876]   File "/usr/lib/python2.7/subprocess.py", line 394, in __init__
W0128 07:00:33.877]     errread, errwrite)
W0128 07:00:33.877]   File "/usr/lib/python2.7/subprocess.py", line 1047, in _execute_child
W0128 07:00:33.877]     raise child_exception
W0128 07:00:33.877] OSError: [Errno 2] No such file or directory
```

Reply:
Sounds good, thanks for checking!